templates/tfengine/components/cicd/manual/configs/tf-apply.yaml

```diff
@@ -15,63 +15,63 @@
 timeout: 21600s
 
 steps:
-  - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:02b06198f1da423183937b60493bdaa20dedf36b1a852a1d7fbb5a537fd943fd"
+  - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:47db3e958fbaa0d95881cf99501ebf4522a261ddc68d9566b7cf70e26cf7cddb"
```

Comment:
Add the terraform version as a comment here, I feel like it's important to point out.

Reply:
Added a note at the top.